anndata/tests/test_views.py

```diff
@@ -545,12 +545,12 @@ def test_invalid_scalar_index(adata, index):
         _ = adata[index]
 
 
-@pytest.mark.parametrize("obs", [False, True])
+@pytest.mark.parametrize("axis", ["obs", "vars"])
 @pytest.mark.parametrize("index", [-100, -50, -1])
-def test_negative_scalar_index(adata, index: int, obs: bool):
-    pos_index = index + (adata.n_obs if obs else adata.n_vars)
+def test_negative_scalar_index(adata, axis, index):
+    pos_index = index + getattr(adata, f"n_{axis}")
```

Comment:
I'm not sure the changes here make the test better. Maybe lets revert?

Reply:
I just need to stop doing aesthethic things like this in unrelated PR and do them independently